### PR TITLE
Massively Nerfs Snowdin Gamer Loot

### PR DESCRIPTION
**Pull request by Cheshify**
## About The Pull Request
- Entirely Removes The Rare Antag Loot
- Rebalances existing antag loot tables
- Removes all but one templar from the map (no more dumb holy swords)
- Removes some syndicate bodies
- Removes some guns
- Swaps some enemies around so there's more plasma miners (spooky!!!)
- added new stuff to replace gamer loot 
- No Xmas Tree w/ presents.

## Why It's Good For The Game
Having people gamer gear up on snowdin and then go back to station with 400 guns and 4 fully equipped syndicate modsuits suck.

## Changelog

:cl:
balance: Snowdin.dmm is now in line with the design doc
/:cl:

### CODE REVIEW

**Comment by DaydreamIQ** [comment]
**Praise be**
No more gamer assistants coming out of the gateway with a CR20, a full kit of spells and enough holy swords to arm the entire station if a cult shows up.

File: code/game/objects/effects/spawners/random/exotic.dm [hunk 1]
```diff
@@ -48,67 +48,29 @@
...
+		/obj/item/shield/riot = 12,
+		/obj/item/stack/sheet/mineral/diamond{amount = 15} = 5,
+		/obj/item/stack/sheet/mineral/uranium{amount = 15} = 5,
+		/obj/item/stack/sheet/mineral/plasma{amount = 15} = 5,
+		/obj/item/stack/sheet/mineral/gold{amount = 15} = 5,
+		/obj/item/grenade/clusterbuster/smoke = 15,
+		/obj/item/clothing/under/chameleon = 13,
+		/obj/item/knife/combat = 10,
+		/obj/item/implantcase/deathrattle = 5,
+		/obj/item/storage/fancy/cigarettes/cigpack_syndicate = 1,
```
Comment (Fikou) [inline code/game/objects/effects/spawners/random/exotic.dm:73]: there is 1 antag item in this spawner called antag loot

**Comment by private-tristan** [comment]
I feel like the smoke book, combat defib, telescopic riotshield, borg self repair board, summon item spellbook should stay in because they're things you can get with enough research (other than the spellbooks, which are mostly misc spells) 

and maybe give BoH a 1% spawn chance because funny👀 
I would have advocated for the syndie medkit but then I realized it has the combat hypospray which heals big good. (maybe some way to have it without the hypo?)

**Comment by Cheshify** [comment]
> I feel like the smoke book, combat defib, telescopic riotshield, borg self repair board, summon item spellbook should stay in because they're things you can get with enough research (other than the spellbooks, which are mostly misc spells)
> 
> and maybe give BoH a 1% spawn chance because funny👀 I would have advocated for the syndie medkit but then I realized it has the combat hypospray which heals big good. (maybe some way to have it without the hypo?)

I would be ok with the tele shield, but the combat defib, medikit, and summon item book are all gear that is INCREDIBLY gamer on-station.

**Comment by MMMiracles** [comment]
as the guy who made snowdin, if you downvote this pr you have the big cringe

**Comment by Cheshify** [comment]
> as the guy who made snowdin, if you downvote this pr you have the big cringe

What's with me always working on your awesome projects to bring them to modern TG

**Comment by IndieanaJones** [comment]
> as the guy who made snowdin, if you downvote this pr you have the big cringe

Have you seen Tram?  You're off your rocker, old man.

**Comment by ATH1909** [comment]
> > I feel like the smoke book, combat defib, telescopic riotshield, borg self repair board, summon item spellbook should stay in because they're things you can get with enough research (other than the spellbooks, which are mostly misc spells)
> > and maybe give BoH a 1% spawn chance because funny👀 I would have advocated for the syndie medkit but then I realized it has the combat hypospray which heals big good. (maybe some way to have it without the hypo?)
> 
> I would be ok with the tele shield, but the combat defib, medikit, and summon item book are all gear that is INCREDIBLY gamer on-station.

summon item spellbooks are also available as tendril loot?

File: _maps/RandomZLevels/snowdin.dmm [hunk 2]
```diff
@@ -530,7 +530,7 @@
 /turf/open/floor/plating,
 /area/awaymission/snowdin/post/dorm)
 "bP" = (
-/obj/structure/flora/tree/pine/xmas/presents,
```
Comment (ATH1909) [inline _maps/RandomZLevels/snowdin.dmm:533]: HEY

Why the hell didn't you mention the removal of the christmas tree in the PR body?! It's, like, one of the most interesting parts of Snowdin for people who aren't interesting in dungeon diving for gamer loot.

File: code/game/objects/effects/spawners/random/exotic.dm [hunk 3]
```diff
@@ -48,67 +48,29 @@
 		/obj/item/storage/medkit/toxin = 35,
 		/obj/item/storage/medkit/brute = 27,
 		/obj/item/storage/medkit/fire = 27,
-		/obj/item/grenade/clusterbuster/smoke = 15,
-		/obj/item/clothing/under/chameleon = 13,
```
Comment (ATH1909) [inline code/game/objects/effects/spawners/random/exotic.dm:52]: why has a chameleon jumpsuit been deemed too "extreme" to be snowdin loot??

File: code/game/objects/effects/spawners/random/exotic.dm [hunk 4]
```diff
@@ -48,67 +48,29 @@
...
 		/obj/item/storage/toolbox/syndicate = 12,
-		/obj/item/melee/baton/telescopic = 12,
-		/obj/item/melee/baton = 11,
-		/obj/item/book/granter/spell/smoke = 10,
-		/obj/item/book/granter/spell/blind = 10,
-		/obj/item/clothing/shoes/chameleon/noslip = 10,
-		/obj/item/grenade/c4 = 7,
 		/obj/item/borg/upgrade/ddrill = 3,
-		/obj/item/borg/upgrade/soh = 3,
```
Comment (ATH1909) [inline code/game/objects/effects/spawners/random/exotic.dm:61]: this is literally station-printable hello???

File: code/game/objects/effects/spawners/random/exotic.dm [hunk 5]
```diff
@@ -48,67 +48,29 @@
 		/obj/item/storage/medkit/toxin = 35,
 		/obj/item/storage/medkit/brute = 27,
 		/obj/item/storage/medkit/fire = 27,
-		/obj/item/grenade/clusterbuster/smoke = 15,
-		/obj/item/clothing/under/chameleon = 13,
 		/obj/item/storage/toolbox/syndicate = 12,
-		/obj/item/melee/baton/telescopic = 12,
-		/obj/item/melee/baton = 11,
-		/obj/item/book/granter/spell/smoke = 10,
-		/obj/item/book/granter/spell/blind = 10,
```
Comment (ATH1909) [inline code/game/objects/effects/spawners/random/exotic.dm:57]: these books were great for pranks, I'd hate to see them go

File: _maps/RandomZLevels/snowdin.dmm (hunk 2 above)
Comment (MMMiracles) [reply _maps/RandomZLevels/snowdin.dmm:533]: okay this part is lame please fix

File: code/game/objects/effects/spawners/random/exotic.dm [hunk 6]
```diff
@@ -48,67 +48,29 @@
...
-		/obj/item/stack/sheet/mineral/gold{amount = 15} = 10,
-		/obj/item/grenade/spawnergrenade/spesscarp = 7,
-		/obj/item/melee/energy/sword = 7,
-		/obj/item/borg/upgrade/disablercooler = 7,
-		/obj/item/dnainjector/lasereyesmut = 7,
-		/obj/item/shield/energy = 6,
-		/obj/item/pickaxe/drill/diamonddrill = 6,
-		/obj/item/grenade/spawnergrenade/manhacks = 6,
-		/obj/item/defibrillator/compact = 6,
-		/obj/item/book/granter/spell/barnyard = 4,
```
Comment (ATH1909) [inline code/game/objects/effects/spawners/random/exotic.dm:87]: this one was FUNNY

**Comment by Shadowflame909** [comment]
Should add more useless gamer gear like a chameleon gun. Very fun but very useless.

File: code/game/objects/effects/spawners/random/exotic.dm [hunk 7]
```diff
@@ -48,67 +48,29 @@
...
+		/obj/item/clothing/glasses/night = 10,
+		/obj/item/pickaxe/drill/diamonddrill = 6,
 	)
 
 /obj/effect/spawner/random/exotic/antag_gear
 	name = "antag gear"
 	icon_state = "esword"
 	loot = list(
-		/obj/item/storage/medkit/tactical = 35,
-		/obj/item/book/granter/spell/summonitem = 20,
```
Comment (ATH1909) [inline code/game/objects/effects/spawners/random/exotic.dm:69]: summon item is already lavaland loot

File: code/game/objects/effects/spawners/random/exotic.dm [hunk 8]
```diff
@@ -48,67 +48,29 @@
...
-		/obj/item/book/granter/spell/forcewall = 17,
-		/obj/item/pneumatic_cannon = 15,
-		/obj/item/book/granter/spell/knock = 15,
```
Comment (ATH1909) [inline code/game/objects/effects/spawners/random/exotic.dm:72]: forcewall and knock aren't too disruptive in the hands of a non-antag, I don't see why we can't let people earn them

File: code/game/objects/effects/spawners/random/exotic.dm [hunk 9]
```diff
@@ -48,67 +48,29 @@
...
-		/obj/item/storage/backpack/holding = 12,
```
Comment (ATH1909) [inline code/game/objects/effects/spawners/random/exotic.dm:73]: this is station-researchable yet valuable and is unrelated to combat; why was this removed?

File: code/game/objects/effects/spawners/random/exotic.dm [hunk 10]
```diff
@@ -48,67 +48,29 @@
...
-		/obj/item/gun/magic/wand/fireball/inert = 3,
-	)
 
-/obj/effect/spawner/random/exotic/antag_gear_strong
-	name = "antag gear strong"
-	icon_state = "esword_dual"
-	loot = list(
-		/obj/item/pickaxe/drill/jackhammer = 30,
-		/obj/item/singularityhammer = 25,
-		/obj/item/fireaxe = 25,
```
Comment (ATH1909) [inline code/game/objects/effects/spawners/random/exotic.dm:98]: c'mon, man

there are literally two of these available on the station at roundstart, why are they getting removed from the loot pool?

File: code/game/objects/effects/spawners/random/exotic.dm [hunk 11]
```diff
@@ -48,67 +48,29 @@
...
-		/obj/item/organ/heart/demon = 7,
-		/obj/item/uplink/old = 2,
+		/obj/item/clothing/glasses/science/night = 15,
+		/obj/item/shield/riot = 12,
+		/obj/item/stack/sheet/mineral/diamond{amount = 15} = 5,
+		/obj/item/stack/sheet/mineral/uranium{amount = 15} = 5,
+		/obj/item/stack/sheet/mineral/plasma{amount = 15} = 5,
+		/obj/item/stack/sheet/mineral/gold{amount = 15} = 5,
+		/obj/item/grenade/clusterbuster/smoke = 15,
+		/obj/item/clothing/under/chameleon = 13,
```
Comment (ATH1909) [inline code/game/objects/effects/spawners/random/exotic.dm:70]: chameleon jumpsuits really aren't that exciting without other chameleon gear to go with them; they generally just get concealed by your other worn equipment

**Comment by Shadowflame909** [comment]
Don't dangerous gateways trigger late into the round now too?

Most of the gamer gear ATH listed can't really compare to late-game extended tech like robotics, or xenobio, or botany.

**Comment by private-tristan** [comment]
> I would be ok with the tele shield, but the combat defib, medikit, and summon item book are all gear that is INCREDIBLY gamer on-station.

unless im misremembering, combat defib is literally just compact defib but EMP'D already, right? 
medkit understandable, the hypo is strong
summon item book is obtainable by miners from tendril loot

**Comment by Cheshify** [comment]
> unless im misremembering, combat defib is literally just compact defib but EMP'D already, right? medkit understandable, the hypo is strong summon item book is obtainable by miners from tendril loot

the medkits CONTAIN combat defibs, and summon item makes you uncontainable by security

**Review by Mothblocks — APPROVED** [review]
Very good, I was wondering how long I would've need to keep asking 😅

**Comment by timothymtorres** [comment]
I was the one who refactored all the mapping random spawners about a year ago.  The antag gear spawners were only used in snowdin (pretty sure) and at first I was considering removing it entirely since it was a niche spot.  It's kind of overkill having a spawner be used only in a single spot.

I'd support removing the antag spawners entirely, but I would recommend you replace it with other cool stuff.  (also change the name to something more befitting)

**Comment by Shadowflame909** [comment]
Yeah replace the gamer loot with crew level gamer loot. That might be harder to get like reactive armor. Or credits. Or a durathread jumpsuit. Rewards can be weak but they can still be rewarding. Miner mats and drills aren't that useful at all

**Comment by ATH1909** [comment]
> the medkits CONTAIN combat defibs, and summon item makes you uncontainable by security

you can still be cuffed, strapped to a bed, straightjacketed, etc. if that still isn't enough for you to be contained for long enough to serve your sentence, you really ought to just be borged or given a chemical implant like other "uncontainable" criminals. I see no difference between this and, say, a toolset implant. act like an uncontainable antag, get treated like one.

**Comment by NoxVS** [comment]
> > > I feel like the smoke book, combat defib, telescopic riotshield, borg self repair board, summon item spellbook should stay in because they're things you can get with enough research (other than the spellbooks, which are mostly misc spells)
> > > and maybe give BoH a 1% spawn chance because funny👀 I would have advocated for the syndie medkit but then I realized it has the combat hypospray which heals big good. (maybe some way to have it without the hypo?)
> > 
> > 
> > I would be ok with the tele shield, but the combat defib, medikit, and summon item book are all gear that is INCREDIBLY gamer on-station.
> 
> summon item spellbooks are also available as tendril loot, no?

Saying that you can get it from mining and thus it is balanced is hardly the argument you think it is.

**Comment by anclandor** [comment]
> 
> the medkits CONTAIN combat defibs, and summon item makes you uncontainable by security

It doesn't really make you uncontainable by security though.  If you have a bag full of tools as your summon, they can just take the tools out.  Even if you have a fireaxe you will end up shocking yourself breaking out, and if they are creative they can mute you to stop you from casting, sit you next to a portable flasher so you are constantly stunned for you sentence time, or just kill you.  Having a toolset implant is arguably better at making you "uncontainable" than summon item.

**Comment by ATH1909** [comment]
> Saying that you can get it from mining and thus it is balanced is hardly the argument you think it is.

I'm saying that because miners generally haven't caused major disruptions to the station with it despite having access to it, it's not an inherently dangerous/toxic item and away mission explorers generally shouldn't end up causing major disruptions to the station with it either.

File: code/game/objects/effects/spawners/random/exotic.dm (hunk 1 above)
Comment (Mothblocks) [reply code/game/objects/effects/spawners/random/exotic.dm:73]: lol, should probably rename it yeah

**Comment by Mothblocks** [comment]
> **DO NOT** Provide the player with obscenely powerful loot. Gateways are part of the normal round progression under the proposed system and this means they should not completely derail the rounds. Some reasonable weapons are fine (think what the player can get at cargo), but think very carefully before handing the player nuke ops gear, as an example.

Gateway guidelines https://hackmd.io/@tgstation/SyYb_cjQ9

**Comment by Cheshify** [comment]
In my opinion, if you're only clearing a gateway for gamer loot, and not for an opportunity to clear it with friends, learn some Common Core lore, build a cool FOB, gather resources/items for the station, or just making some simplemobs horizontal, then you should stay out of gateways.

**Comment by LemonInTheDark** [comment]
Kinda sad to see the tree go, it's one of those chaotic parts of the game that makes me kinda happy

**Comment by MMMiracles** [comment]
bring the tree back coward

**Comment by Shadowflame909** [comment]
> In my opinion, if you're only clearing a gateway for gamer loot, and not for an opportunity to clear it with friends, learn some Common Core lore, build a cool FOB, gather resources/items for the station, or just making some simplemobs horizontal, then you should stay out of gateways.

Gateway should still give you tools to survive itself though.

You'd have to powergame pretty hard on station to go into the gateway and fight hordes of monsters.

**Review by Maurukas — APPROVED** [review]
We tried to have a discussion about this - and unfortunately it got rather unpleasant. Everyone has apologized now and I think we're fine. 

The loot spawners must go. They're over the top by any standards.

The problem with the Christmas tree is that it can inject some very bad items into a round. The worst offender is the debug uplink, but it can pull any debug item as a gift. This is incompatible with the design doc. 

The good thing about the tree is that it injects a lot of randomness into the round in terms of items that are unusual or unobtainable normally. Players like this. You can have whole gimmicks around an item just because it's rare. It doesn't have to be a gun - it can be a funny hat, a uniform, a sleeper board. Anything that is not usually seen. None of these things derail a round, and that's most of what comes from the tree. As lemoninthedark mentions - this is a source of the good chaos. 

So here's what's going to happen. We'll merge this. @mmmiracles has said that he's willing to make a subtype of the tree for gateway use that excludes the worst stuff (debug and admin items at a minimum). When he's finished with that, Snowdin gets its Christmas tree back.
